chord_metadata_service/patients/tests/test_api.py

```diff
@@ -561,13 +562,15 @@ def test_public_filtering_extra_properties_multiple_ranges_1(self):
     @override_settings(CONFIG_PUBLIC=CONFIG_PUBLIC_TEST)
     def test_public_filtering_extra_properties_date_range_1(self):
         # extra_properties date range search (only after or before, single value)
+        # Testing with a date of consent from 1 year ago
+        year_of_consent = datetime.now().year - 1
```

Comment:
minor: i think i'd prefer if all dates were fixed rather than being based on now() (so basically changing the other side of this test, the test data rather than the test data itself)